lib/utils.js

```diff
@@ -168,7 +168,7 @@ var encode = function encode(str) {
 };
 
 var compact = function compact(value) {
-    var queue = [{ obj: { o: value }, prop: 'o' }];
+    var queue = [{ obj: { 'o': value }, prop: 'o' }]; // eslint-disable-line quote-props
```

Comment:
If this breaks closure compiler in advanced mode, it's because closure compiler is broken in advanced mode. This is a valid unquoted property key, and it's not acceptable for a tool to require this kind of pollution to be pushed upstream.

I'd suggest using a non-broken bundling tool instead.

Reply:
Got it - sorry to bother you. I'll go find a different library.